src/content/building/storage.md

```diff
@@ -1,3 +1,9 @@
+---
+previousText: 'Webview'
```

Comment:
@janpio Could we use 'Web View' as `previousText` here to match the page title?

Reply:
Sure, changed.